core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1350,14 +1321,14 @@ public Api getApi() {
     /**
      * The file we save our configuration.
      */
-    private static @NonNull XmlFile getConfigFile(@NonNull File file) {
```

Comment:
It does not look to be a good public API. Why would you need it here? Could it be just moved to the storage?